Modules/_testcapimodule.c

```diff
@@ -1016,6 +1016,31 @@ test_buildvalue_N(PyObject *self, PyObject *Py_UNUSED(ignored))
     Py_RETURN_NONE;
 }
 
+static PyObject *
+test_buildvalue_ignore_char(PyObject *self, PyObject *Py_UNUSED(ignored))
+{
+    PyObject *res = Py_BuildValue("(i )", 0);
```

Comment:
Please add also tests for list and dict.

Reply:
Okay.